.github/actions/python-build-vsix/action.yml

```diff
@@ -0,0 +1,104 @@
+name: 'Build VSIX'
+description: "Build the extension's VSIX"
+
+inputs:
+  node_version:
+    description: 'Version of Node to install'
+    required: true
+  vsix_name:
+    description: 'Name to give the final VSIX'
+    required: true
+  artifact_name:
+    description: 'Name to give the artifact containing the VSIX'
+    required: true
+  cargo_target:
+    description: 'Cargo build target for the native build'
+    required: true
+  vsix_target:
+    description: 'vsix build target for the native build'
+    required: true
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Install Node
+      uses: actions/setup-node@v4
+      with:
+        node-version: ${{ inputs.node_version }}
+        cache: 'npm'
+
+    - name: Rust Tool Chain setup
+      uses: dtolnay/rust-toolchain@stable
+
+    # Jedi LS depends on dataclasses which is not in the stdlib in Python 3.7.
+    - name: Use Python 3.8 for JediLSP
+      uses: actions/setup-python@v5
+      with:
+        python-version: 3.8
+        cache: 'pip'
+        cache-dependency-path: |
+          extensions/positron-python/requirements.txt
```

Comment:
is there a way to add a working directory for files in `.github/actions/`? It doesn't look like the classic

```
defaults:
 run:
  working-directory: 'extensions/positron-python'
```

works as it does in files under `.github/workflows`